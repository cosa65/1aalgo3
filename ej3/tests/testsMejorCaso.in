a ;b ;c
a ;b ;c ;d
a ;b ;c ;d ;e
a ;b ;c ;d ;e ;f
a ;b ;c ;d ;e ;f ;g
a ;b ;c ;d ;e ;f ;g ;h
a bc;b ac;c ab
a bcd;b acd;c abd;d abc
a bcde;b acde;c abde;d abce;e abcd
a bcdef;b acdef;c abdef;d abcef;e abcdf
a bcdefg;b acdefg;c abdefg;d abcefg;e abcdfg;f abcdeg;g abcdef
a bcdefgh;b acdefgh;c abdefgh;d abcefgh;e abcdfgh;f abcdegh;g abcdefh;h abcdefg
