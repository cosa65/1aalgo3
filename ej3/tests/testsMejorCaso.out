0 abc
0 abcde
0 abcdefg
